src/services/repo/useActivateFlagMeasurements.tsx

```diff
@@ -0,0 +1,111 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query'
+import z from 'zod'
+
+import { renderToast } from 'services/toast'
+import Api from 'shared/api'
+import { NetworkErrorObject } from 'shared/api/helpers'
+
+export const MEASUREMENT_TYPE = {
+  FLAG_COVERAGE: 'FLAG_COVERAGE',
+  COMPONENT_COVERAGE: 'COMPONENT_COVERAGE',
+} as const
+
+interface ActivateFlagMeasurementsProps {
+  provider: string
+  owner: string
+  repo: string
+  measurementType: (typeof MEASUREMENT_TYPE)[keyof typeof MEASUREMENT_TYPE]
```

Comment:
nit: maybe we could pull this out into a type separately as `export type MeasurementType = keyof typeof MEASUREMENT_TYPE` so if we need to use it later we can. Also aids a bit with readability here imo